www/index.html.tmpl

```diff
@@ -39,7 +44,16 @@
                     text = '',
                     values = {};
 
+                var preventSelection = function(e) {
+                    e.preventDefault();
+                }
```

Comment:
missing semicolon

(ok, technically you can omit them in *most* cases in JS, but most style guides recommend against it)

Reply:
Nice, thanks for catching that.